### PR TITLE
Update config.json

### PR DESCRIPTION
**Pull request by pentoo1989**
fix error on startup "fatal: can only run as pid 1 "
S6-Overlay 3.x update on our docker base images

https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images/

### CODE REVIEW

File: zigbee2mqtt-edge/config.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "Zigbee2MQTT Edge",
   "version": "edge",
+  "Init": true,
```
Comment (Koenkk) [inline zigbee2mqtt-edge/config.json:4]: It is already there: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/blob/109fe6bb73acc7af1c2d6c59feac11a63119c71b/zigbee2mqtt-edge/config.json#L20

Comment (pentoo1989) [reply zigbee2mqtt-edge/config.json:4]: sorry, didnt find that
but it has to be true not false

Comment (Koenkk) [reply zigbee2mqtt-edge/config.json:4]: @mdegat01 can you confirm this has to be `true` and not false as you commited in https://github.com/zigbee2mqtt/hassio-zigbee2mqtt/commit/62f74aade44e24718ffbeb119d1fe110b6c9c918 ?

Comment (pentoo1989) [reply zigbee2mqtt-edge/config.json:4]: for me it works

**Pull request closed without merging**